consensus/scp/src/slot.rs

```diff
@@ -1332,9 +1324,9 @@ impl<V: Value, ValidationError: Display> Slot<V, ValidationError> {
     }
 
     /// "Accepted Nominated" values that are not yet in self.Y.
-    fn additional_values_accepted_nominated(&self) -> BTreeSet<V> {
+    fn additional_values_accepted_nominated(&self) -> HashSet<V> {
         // 1) Find values that can be accepted because a blocking set has issued accept nominate.
-        let mut accepted_from_blocking_set: BTreeSet<V> = {
+        let accepted_from_blocking_set: BTreeSet<V> = {
```

Comment:
These predicates still use BTreeSet (instead of HashSet). I don't see why BTreeSet is necessary, but changing it to HashSet breaks some of the crate integration tests. MC-1444 is about investigating this.